libnetdata/config/appconfig.c

```diff
@@ -340,6 +340,32 @@ long long appconfig_get_number(struct config *root, const char *section, const c
     return strtoll(s, NULL, 0);
 }
 
+long long appconfig_get_number_min_max(struct config *root, const char *section, const char *name, long long value, long long min, long long max)
```

Comment:
It looks an overkill to add a new function min_max  just to check that we need at least 1 query thread up to a max defined

Reply:
well it seems like common operation to have some config values where we want ensure min and max values

Reply:
And yet it has never occurred in the hundreds of config options that we currently have. This is because we try to avoid placing arbitrary limits.